packages/node_modules/@webex/internal-plugin-search/src/search.js

```diff
@@ -59,9 +59,11 @@ const Search = WebexPlugin.extend({
           searchEncryptionKeyUrl: this.webex.internal.device.searchEncryptionKeyUrl
         })
       }))
-      .then((res) => get(res, 'body.activities.items') || []);
+      .then((res) => ({
+        activities: get(res, 'body.activities.items', []),
```

Comment:
Instead of introducing a breaking change in the response for this, could we instead make breadcrumbs an item in the `options` object passed to `search(`? That way, sdk users can opt-in?

Reply:
okay sure no problem